protocol/avdl/metadata.avdl

```diff
@@ -33,4 +39,12 @@ protocol metadata {
   bytes getFolderHandle(string folderID, string signature, string challenge);
   void getFoldersForRekey(KID deviceKID);
   void ping();
+
+  // Merkle tree specific commands
+  // todo: we probably also want a method which will return a minmum set of interior nodes
+  // and a leaf node given a root node and a TLF ID.
+  MerkleRoot getMerkleRoot(string tree, long seqNo);
+  MerkleRoot getMerkleRootLatest(string tree);
+  MerkleRoot getMerkleRootSince(string tree, long when);
```

Comment:
I believe it's # of milliseconds since the epoch, but there are methods in `extras.go` to convert back and forth to Go times.